web/src/PickNeighbourhoodMode.svelte

```diff
@@ -160,36 +168,88 @@
   </div>
 
   <div slot="sidebar">
-    <div><Link on:click={newBoundary}>Draw a new boundary</Link></div>
-    <div>
-      <Link on:click={() => ($mode = { mode: "auto-boundaries" })}>
-        Use an auto-generated boundary
-      </Link>
-    </div>
+    <h3 style="padding: 4px 8px; margin-bottom: 0">Neighbourhoods</h3>
+    <ul class="neighbourhood-list">
+      {#each neighbourhoods.features as { properties: { name, area_km2, simd } }}
+        <!-- 
+          TODO: use hoveredNeighbourhoodFromList to highlight the feature on the map.
+          It's tricky, because we have to plumb it through to 
+          maplibre somehow. I tried using feature-state via <JoinedData>, but failed.
+          The proximate error was that joining data requires referencing a source-id,
+          but maplibre complains when you try to set the feature state of a GeoJSON layer 
+          with a source-id.
 
-    <ul>
-      {#each boundaryNames as name}
-        <li>
+          Maybe dustin has a clever idea?
+        -->
+        <li
+          on:mouseenter={() => (hoveredNeighbourhoodFromList = name)}
+          on:mouseleave={() => (hoveredNeighbourhoodFromList = null)}
+          class:highlighted={hoveredMapFeature?.properties.name == name}
+        >
           <span style="display: flex; justify-content: space-between;">
-            <Link on:click={() => pickNeighbourhood(name)}>
-              {name}
-            </Link>
-            <button
-              class="secondary"
-              on:click={() => renameNeighbourhood(name)}
-            >
-              <img src={editIcon} alt="Rename neighbourhood" />
-            </button>
-            <button
-              class="secondary"
-              on:click={() => deleteNeighbourhood(name)}
-            >
-              <img src={deleteIcon} alt="Delete neighbourhood" />
-            </button>
+            <span class="neighbourhood-name">
```

Comment:
FYI - I'm making some pretty broad changes to this component, so try to avoid it for a bit if you can to minimize conflicts.